src/Auth0.OidcClient.WPF/PlatformWebView.cs

```diff
@@ -10,10 +10,25 @@ namespace Auth0.OidcClient
     public class PlatformWebView : IBrowser
     {
         private readonly Func<Window> _windowFactory;
+        private readonly bool _shouldCloseWindow;
 
-        public PlatformWebView(Func<Window> windowFactory)
+        /// <summary>
+        /// Constructor which lets you pass in your WPF window and window closing property
+        /// </summary>
+        /// <param name="windowFactory"> </param>
+        /// <param name="shouldCloseWindow"> Determines whether the window closes or not after sucessful login</param>
+        
+        /// Example Usage:
```

Comment:
There is actually an XML doc tag for this - https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/example

Reply:
Good to know. Updated it with the right tags.